ansible/roles/common_installs/tasks/main.yml

```diff
@@ -58,6 +58,7 @@
     - libfreetype6-dev
     - gradle
     - ranger  # cmdline file browser
+    - libffi-dev  # Required for installing github3py
```

Comment:
This task is installing lots of stuff that is not needed on many machines. For example, `libffi-dev` (and many others too) is not needed by Riak CS machines. Not a blocker for this PR, but would be nice if we moved a lot of this stuff to the `commcarehq` role.